tsconfig.json

```diff
@@ -21,7 +21,7 @@
     "strict": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "baseUrl": "./src",
+    "baseUrl": ".",
```

Comment:
What does this do? Looks unrelated?

Reply:
This is not related to WKB, but it seems not been set correctly and causing the alias "@kepler.gl/*" not mapping to correct source code. (see below `@kepler.gl/*": ["src/*/src"]`)